Dockerfile

```diff
@@ -6,103 +6,131 @@ ARG NCPUS=1
+RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh \
+    && /bin/bash ~/miniconda.sh -b -p /opt/conda \
+    && rm ~/miniconda.sh \
+    && /opt/conda/bin/conda clean -tipsy \
+    && ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh \
+    && echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc
 
-RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh -O ~/miniconda.sh && \
-  /bin/bash ~/miniconda.sh -b -p /opt/conda && \
-  rm ~/miniconda.sh && \
-  /opt/conda/bin/conda clean -tipsy && \
-  ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh && \
-  echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc
-#
-#
 # Configure R
 COPY files/.condarc /opt/conda/
 RUN conda install python=$PY_VERSION r-base=$R_VERSION
 RUN echo '.libPaths(c("~/R/library",paste0(R.home(), "/library"), .libPaths()))' >> /usr/local/lib/R/etc/Rprofile.site \
     && echo "PATH=\"${PATH}\"" >> /usr/local/lib/R/etc/Renviron \
     && echo "AWS_DEFAULT_REGION=eu-west-1" >> /usr/local/lib/R/etc/Renviron \
-    && echo "r-libs-user=~/R/library" >> /etc/rstudio/rsession.conf \
-## Users want nano when they git commit not vim
-## Users will use the reticulate package within R to run Python code.  This makes sure we always use py3
-    && update-alternatives --set editor /bin/nano
+    && echo "r-libs-user=~/R/library" >> /etc/rstudio/rsession.conf
 
-#
-#
 # Install Conda Packages
-COPY files/conda_packages /tmp/
 RUN conda config --system --set pip_interop_enabled True \
     && conda install \
-    $(cat /tmp/conda_packages) \
-    && rm -f /tmp/{apt_packages,conda_packages}
-
+    boto3 \
+    r-aws.s3 \
+    r-aws.ec2metadata \
+    r-Rcpp \
+    r-base64enc \
+    r-bitops \
+    r-caTools \
+    r-codetools \
+    r-curl \
+    r-devtools \
+    r-digest \
+    r-evaluate \
+    r-formatR \
+    r-highr \
+    r-htmltools \
+    r-httr \
+    r-jsonlite \
+    r-knitr \
+    r-markdown \
+    r-packrat \
+    r-readr \
+    r-reticulate \
+    r-rmarkdown \
+    r-rprojroot \
+    r-shiny \
+    r-stringr \
+    r-s3tools \
+    r-tidyverse \
+    r-webshot \
+    r-xml2 \
+    r-yaml \
+    giflib \
+    cairo \
+    pango \
+    libjpeg-turbo \
+    nodejs \
+    altair \
+    libiconv \
+    nbstripout
+
+# Install phantomjs via webshot for Doc/PDF with JS graphs in it
 RUN ln -s /bin/tar /bin/gtar \
-#    # Install phantomjs via webshot for Doc/PDF with JS graphs in it
     && R -e "library(webshot); webshot::install_phantomjs()" \
     && mv /root/bin/phantomjs /usr/bin/phantomjs \
     && chmod a+rx /usr/bin/phantomjs
 
 RUN npm config set unsafe-perm true \
     && npm install -g vega vega-lite
 
-# We want all packages to use the default MRAN mirror so that when we upgrade users, they don't magically get new packages
+# We want all packages to use the default MRAN mirror so that when we upgrade users, they dont magically get new packages
```

Comment:
interesting how the `don't` was changed to `dont` 😬

Reply:
:D what now, fixed